File: python_programming/practices/time_of_day.py
# HW 7th time of day


import datetime

now = datetime.datetime.now()
hour = now.hour

print(f"The hour is {hour}")

if hour >= 17:
    print("Good evening!")
elif hour >= 12:
    print("Good afternoon!")
elif hour >= 7:
    print("Good morning!")
else:
    print("Good night!")

File: c_programming/practices/time_of_day.c
//HW 7th time of day
#include <stdio.h>


int main(void){
    int hour;
    printf("What time is it in military time: ");
    scanf("%d", &hour);

     } if (hour < 12) {
        printf("Good Morning!\n");
    } else if (hour < 18) {
        printf("Good Afternoon!\n");
    } else {
        printf("Good Evening!\n");
    }

    return 0;
}